README.md

```diff
@@ -32,6 +32,43 @@
 
 [English Docs](/README_en.md)
 
+# 支持的环境
+
+## Linux 发行版, CPU架构
+
+- Ubuntu 16.04， 18.04， 20.04 ,  x86_64/ arm64
+- Centos/RHEL 7.6+,  x86_64/ arm64
+- 其他支持 systemd 的系统环境.  x86_64/ arm64
+
+## kubernetes 版本
+
+- 1.16+
+- 1.17+
+- 1.18+
+- 1.19+
+- 1.20+
+
+更多版本支持, 详细查看[sealyun.com](https://www.sealyun.com)
+
+## 要求和建议
+
+- 最低资源要求
+   - 2 vCpu
+   - 4G Ram
+   - 40G+ 存储
+
+- 操作系统要求
+   - ssh 可以访问各安装节点
+   - 各节点主机名不相同，并满足kubernetes的主机名要求。
+   - 各节点时间同步
+   - 网卡名称如果是不常见的，建议修改成规范的网卡名称， 如(eth.*|en.*|em.*)
```

Comment:
fannel 也是支持ip的。
```
--iface-regex="": regex expression to match the first interface to use (IP or name) for inter-host communication. 
If unspecified, will default to the interface for the default route on the machine. 
This can be specified multiple times to check each regex in order. 
Returns the first match found. This option is superseded by the iface option and will only be used if nothing matches any option specified in the iface options.
```

calico： 
```
 can-reach=192.168.0.1 
```

网关的写法有点问题， flannel的和 calico还不一样。。。得紧急修复一下